pkg/imageregistry/imageregistry.go

```diff
@@ -0,0 +1,328 @@
+package imageregistry
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/apiservers"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/clusteroperator"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	imageregistryv1 "github.com/openshift/api/imageregistry/v1"
+	v1 "github.com/openshift/api/operator/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	goclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+var imageRegistryObjName string = "cluster"
```

Comment:
Please make it func arguments. Let's don't hard code these parameters

Reply:
it is constant, it can not changed
I can add it to the constants, but I'm not sure it is make any sence